dist/redhat/scylla.spec

```diff
@@ -21,6 +21,10 @@ Obsoletes:	scylla-server < 1.1
 %undefine _unique_build_ids
 %global _no_recompute_build_ids 1
 
+%ifnarch x86_64
+%global _missing_build_ids_terminate_build 0
+%endif
```

Comment:
We'll have to fix it. Please file an issue so we don't forget and copy @espindola. Meanwhile we can live without the build-id.

Reply:
made #7046, also find out patchelf breaking build-id